dtlpymetrics/utils/metrics.py

```diff
@@ -366,13 +366,19 @@ def general_match(matches: Matches,
             first_annotation = [a for a in first_set if a.id == first_annotation_id][0]
             second_annotation = [a for a in second_set if a.id == second_annotation_id][0]
             geometry_score = df.iloc[row_index, col_index]
-            labels_score = Matchers.match_labels(label1=first_annotation.label,
-                                                 label2=second_annotation.label)
-            attribute_score = Matchers.match_attributes(attributes1=first_annotation.attributes,
-                                                        attributes2=second_annotation.attributes)
-
-            # TODO use ignores for final score
-            annotation_score = (geometry_score + attribute_score + labels_score) / 3
+            labels_score = None
```

Comment:
not sure this is the right way to go about this. it will fill the results with Nonetypes, so what is the use case in which this would make sense?